leakcanary-android-core/src/main/java/leakcanary/internal/activity/screen/HeapAnalysisFailureScreen.kt

```diff
@@ -25,13 +25,13 @@ internal class HeapAnalysisFailureScreen(
       val pair = HeapAnalysisTable.retrieve<HeapAnalysisFailure>(db, analysisId)
 
       if (pair == null) {
-        activity.title = "Analysis deleted"
+        activity.title = resources.getString(string.leak_canary_analysis_deleted_title)
         return this
       }
 
       val (heapAnalysis, _) = pair
 
-      activity.title = resources.getString(R.string.leak_canary_analysis_failed)
+      activity.title = resources.getString(string.leak_canary_analysis_failed)
```

Comment:
R. gone

Reply:
Ah yeah, I was noticing both imports were available. Do you to prefer to import both `R` and `R.string` or to fully qualify everything? Happy to go back and find all of these in another PR to get it consistent.

Reply:
Yep I'm old school I like my "R.string"s where I use them. Not sure why Android Studio keeps importing the inner classes.